src/driver/mpas.F

```diff
@@ -11,11 +11,14 @@ program mpas
 
    implicit none
 
-   call mpas_init()
+   type (core_type), pointer :: corelist
+   type (domain_type), pointer :: domain_ptr
 
-   call mpas_run() 
+   call mpas_init(corelist,domain_ptr)
```

Comment:
This is nitpicking to be sure, but almost everywhere we use a single space after each comma in an argument list. For consistency, it might be nice to do the same in all new code.